bevy_rapier2d/Cargo.toml

```
@@ -34,7 +46,7 @@ async-collider = ["bevy/bevy_asset", "bevy/bevy_scene"]
 [dependencies]
 bevy = { version = "0.13", default-features = false }
 nalgebra = { version = "0.32.3", features = ["convert-glam025"] }
-rapier2d = "0.19.0"
+rapier2d = "0.20"
```

Should I set explicitly the patch version ?
```suggestion
rapier2d = "0.20.0"
```

---

Not necessarily, that doesn’t make any difference when the patch version is 0.